scripts/build/Jenkins/Jenkinsfile

```diff
@@ -934,6 +983,7 @@ catch(Exception e) {
 finally {
     try {
         node('controller') {
+            ReplaceContent()
```

Comment:
Can we replace content only when build fails? In most cases, people only see logs of failed builds.